### DISCUSSION TITLE
https://github.com/colouring-cities/colouring-london versus https://discuss.colouring.london/

### DISCUSSION THREAD

**Discussion by Toase-of-London** [colouring-cities/colouring-britain#25] [Bug Reports]
I'm sure the menu option "Discussion Forum" on the Main Menu used to link to https://discuss.colouring.london/

It now (well at least on 17-Jul-23 and 18-Jul-23) it links to https://github.com/colouring-cities/colouring-london/discussions

https://discuss.colouring.london/ is a more domain knowledge discussion forum as opposed to a developer forum, and it is the one I wanted. I managed to reach it via the link available are clicking on the "Contact" menu item.

Was this a deliberate change? It seems to make the UI less user friendly.

**Comment by polly64** [comment]
Hi @Toase-of-London yes sorry for confusion we are doing this now so we have a technical and non technical link in the Menu - let us know if any problems once these are up

**Comment by mdsimpson42** [comment]
It was a deliberate change, as the programme has expanded we now have other [Colouring Cities](https://colouringcities.org/) projects starting up. We decided to use GitHub for the discussions on the other "Colouring..." platforms, instead of having to create (and pay for hosting for) a separate Forum page for each of them.

However, for Colouring London, the link to the old Discussion Forum has been restored in the latest version, which is now deployed.
